### ISSUE TITLE
 Training parallelism on multiple machines

### ISSUE THREAD

**Issue by George3d6** [mindsdb/lightwood#64]
We should start thinking about this once we're done with #63 , pytorch and various pytroch related frameworks should provide some support for this. But I doubt it's going to be very useful outside of very large datasets, due to the large data transfer and synchronization overhead.

Good candidate to try this out on would be the Ax Optimizer, which could run multiple trials on different machines.

**Comment by George3d6** [comment]
Removing this, will re-open when we get to the problem, info here is outdated and I wasn't thinking the correct way about it (i.e. I wasn't considering how efficient parallel gradient computations which are accumulated one one machine can actually be... the answer is apparently very).
